warp10/src/main/java/io/warp10/continuum/Configuration.java

```diff
@@ -795,6 +795,11 @@
   public static final String STORE_KAFKA_DATA_GROUPID = "store.kafka.data.groupid";
 
   /**
+   * ConsumerId infix will be affixed inside the Kafka consumerId
+   */
+  public static final String STORE_KAFKA_DATA_CONSUMERID_INFIX = "store.kafka.data.consumerid.infix";
```

Comment:
Please also modify etc/conf-distributed.template to explain the purpose of this configuration